docs/04_web/12_forms/01_foundation-forms.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: 'Foundation Forms'
 sidebar_label: 'Foundation Forms'
-id: 'foundation-forms'
+id: foundation-forms
```

Comment:
@wjhendry do you know if it is ok to remove the quotes around this? The tool seems to have done that for some reason